Source/Blazorise/Components/MessageAlert/MessageAlert.razor.cs

```diff
@@ -305,7 +339,7 @@ protected virtual ModalSize Size
     /// <summary>
     /// Occurs after the user respond with an action button.
     /// </summary>
-    [Parameter] public TaskCompletionSource<bool> Callback { get; set; }
+    [Parameter] public TaskCompletionSource<object> Callback { get; set; }
```

Comment:
I think people are using it for confirmation. 
What about making it generic instead of the object?

Reply:
I have no idea how to make this part more generic TBH.